python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1335,8 +1358,23 @@ def _make_outputs(self, wcsf, visitSummaryTables, exposureInfo):
 
             for d, detector in enumerate(visitSummary["id"]):
                 mapName = f"{visit}/{detector}"
-
-                mapElements = wcsf.mapCollection.orderAtoms(f"{mapName}/base")
+                if mapName in wcsf.mapCollection.allMapNames():
+                    mapElements = wcsf.mapCollection.orderAtoms(f"{mapName}/base")
+                else:
+                    # This extension was not fit, but try to recover WCS
+                    genericElements = mapTemplate["EXPOSURE/DEVICE/base"]["Elements"]
+                    mapElements = []
+                    instrument = visitSummary[0].getVisitInfo().getInstrumentLabel()
+                    for component in genericElements:
```

Comment:
I really don't understand what this nested set of loops is supposed to do. I assume it's a gbdes internals thing?

Reply:
Sort of...this loop goes through the generic definitions for the mappings that take you from pixels to sky and uses those to get the names for the maps for a specific extension (i.e. visit/detector). I spent a lot of time trying to get `gbdes` to do this itself, but couldn't manage it, so this is my replication of how `gbdes` makes the names.